actionview/actionview.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activesupport", version
 
   s.add_dependency "builder",       "~> 3.1"
+  s.add_dependency "erubi",         "~> 1.4"
   s.add_dependency "erubis",        "~> 2.7.0"
```

Comment:
I'll include this change in my next commit, but won't it break things for people who access `ActionView::Template::Handlers::Erubis` and use a Gemfile that doesn't explicitly specify erubis?  Then instead of getting a deprecation message, they get a NameError when they reference the constant.